zanata-war/src/main/java/org/zanata/action/ProjectSlug.java

```diff
@@ -22,16 +22,18 @@
 
 import java.io.Serializable;
 
-import org.apache.deltaspike.core.api.scope.ViewAccessScoped;
+import org.apache.deltaspike.core.api.scope.ConversationGroup;
+import org.apache.deltaspike.core.api.scope.GroupedConversationScoped;
 
 /**
- * We set this to ViewAccessScoped so that it can retain the value accross
- * requests (original request, subsequent ajax requests)
+ * We set this to GroupedConversationScoped so that it can retain the value
+ * across requests (original request, subsequent ajax requests)
```

Comment:
wouldn't `@ViewScope` solve this problem? `@GroupedConversationScoped` should solve this problem across multiple requests for different pages.